src/pages/index.js

```diff
@@ -105,27 +206,104 @@ function getCardElement(data) {
     cardLikeButton.classList.toggle("card__like-button_liked");
   });
 
+  cardLikeButton.addEventListener("click", (evt) => handleLike(evt, data._id));
+
   cardImage.addEventListener("click", () => {
     openModal(previewModal);
     previewModalCaptionEl.textContent = data.name;
     previewModalImageEl.src = data.link;
     previewModalImageEl.alt = data.name;
   });
 
-  cardDeleteButton.addEventListener("click", () => {
-    cardElement.remove();
-  });
+  cardDeleteButton.addEventListener("click", () =>
+    handleDeleteCard(cardElement, data._id)
+  );
 
   return cardElement;
 }
 
+function disableButton(button, settings) {
+  button.disabled = true;
+  if (settings && settings.inactiveButtonClass) {
+    button.classList.add(settings.inactiveButtonClass);
```

Comment:
Please, delete `disableButton` from `index.js`  because you should import the same function from `validation.js` not to duplicate the same code